examples/Attic & Button/app/models/log_message.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+#
+#    Copyright 2018, Optimizely and contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+class LogMessage < ActiveHash::Base
+  @data = []
+  @user_id = nil
+
+  fields :type, :message, :user_id
+  field  :created_at, default: Time.now
+
+  class << self
+    attr_accessor :data
+    attr_accessor :user_id
+  end
+
+  LOGGER_LEVELS = {DEBUG: 0, INFO: 1, WARN: 2, ERROR: 3, FATAL: 4, UNKNOWN: 5}.freeze
+
+  def self.create_record(type, message)
+    @data << create(
+      type: type,
+      message: message,
+      created_at: Time.now,
+      user_id: @user_id
+    )
+  end
+
+  def self.assign_user!(user_id)
+    @user_id = user_id
+  end
+
+  def self.all_logs(user_id)
+    logs = @data.select { |i| i[:user_id] == user_id }
```

Comment:
Add an optional second argument like asc, or desc. And keep one of them as default. And then return records in that order

Reply:
Not addressed.